LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Why this specifically? In the other repos that host the library code, we have [something rather custom](https://github.com/GetStream/stream-chat-android/blob/master/LICENSE) as the LICENSE.

Reply:
I've talked with @tbarbugli and he told me that we can use BSD here